packages/client/tests/functional/metrics/enabled/tests.ts

```diff
@@ -218,7 +218,8 @@ testMatrix.setupTestSuite(
       skipTestIf(
         providerFlavor === ProviderFlavors.JS_NEON ||
           providerFlavor === ProviderFlavors.JS_PG ||
-          providerFlavor === ProviderFlavors.JS_LIBSQL,
+          providerFlavor === ProviderFlavors.JS_LIBSQL ||
```

Comment:
Should the whole test suite be skipped perhaps?

Reply:
Yes, I think we could at this stage, or try to make things more granular?

Reply:
Leaving it granular might make things easier in the future so that we could un-skip them one by one, so no strong preference either way.